src/core/engines/cross-margin/CrossMarginLib.sol

```diff
@@ -149,11 +149,10 @@ library CrossMarginLib {
         } else account.longs[index].amount = newLongAmount;
     }
 
-    ///@dev Settles the accounts short calls and puts, reserving collateral for ITM options
+    ///@dev Settles the accounts longs and shorts
     ///@param account CrossMarginAccount memory that will be updated in-place
```

Comment:
this comment seems wrong: we're updating account storage directly

Reply:
Yep - thats correct. This was copied over from FullMarginLib. It'll be worth opening a dedicated PR for it.